sklearn/multioutput.py

```diff
@@ -0,0 +1,212 @@
+"""
+MultiOneVsRestClassifier
+===========================
+This module includes several classes that extend base estimators to 
+multi-target estimators. 
+
+Most sklearn estimators use a response matrix to train a target function
+with a single output variable.  I.e. typical estimators use the 
+training set X to estimate a target function f(X) that 
+predicts a single Y.  The purpose of this class is to extend estimators
+to be able to estimate a series of target functions (f1,f2,f3...,fn)
+that are trained on a single X predictor matrix to predict a series
+of reponses (y1,y2,y3...,yn).
```

Comment:
this paragraph looks great but it should not belong to an example and not here, I think